components/News/NewsGrid.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+import NewsItem from "./NewsItem";
+import { TypeNews } from "../../generated/contentful";
+import Style from "./styles.module.scss";
+import clsx from "clsx";
+
+export interface NewsGridProps {
+  news: TypeNews[];
+  title?: string;
+  className?: string;
+}
+
+const NewsGrid: FunctionComponent<NewsGridProps> = ({
+  news,
+  title = null,
+  className = "",
+}) => {
+  return (
+    <>
+      {!!title && (
```

Comment:
Shouldn't need to use `!!` to convert to true boolean. A simple condition will catch `null`, `undefined`, or `""` just as well.

 ```suggestion
      {title && (
```